source/_posts/2015-10-17-headless-firefox-in-ubuntu-on-virtual-box-for-cucumber-testing.markdown

```diff
@@ -0,0 +1,78 @@
+---
+layout: post
+title: "Headless firefox in ubuntu on virtual box for cucumber testing"
+date: 2015-10-17 14:12:56 +0200
+comments: true
+author: Ivica Lakatoš
+categories: 
+  - Ruby and Rails
+  - Testing
+tags: 
+  - ruby
+  - rails
+  - vagrant
+  - virtual box
+  - ubuntu
+  - cucumber
+  - selenium-webdriver
+published: true
+---
+
+If you use [Vagrant](http://www.vagrantup.com/downloads.html) and [VirtualBox](https://www.virtualbox.org/) (e.g. some Ubuntu package) to build your Rails app and you want to test it with cucumber scenarios, this is right post for you. 
```

Comment:
@Ikera I think that this might be a good place to mention that they use Ubuntu without an X server and GUI.